pandas/core/internals/managers.py

```diff
@@ -667,6 +675,8 @@ def is_mixed_type(self) -> bool:
 
     @property
     def is_numeric_mixed_type(self) -> bool:
+        # Warning, consolidation needs to get checked upstairs
+        self._consolidate_inplace()
```

Comment:
this one should be pretty obviously innocuous

Reply:
re-reverted